.travis.yml

```diff
@@ -13,10 +13,12 @@ env:
   - SOLC_URL_MACOS='https://www.dropbox.com/s/4amq3on2ds1dq36/solc_0.4.23?dl=0'
   - SOLC_VERSION='v0.4.23'
   matrix:
-  - TEST_TYPE='integration' BLOCKCHAIN_TYPE='--blockchain-type=geth --blockchain-cache'
-  - TEST_TYPE='unit'
-  - TEST_TYPE='long_running' BLOCKCHAIN_TYPE='--blockchain-type=geth --blockchain-cache'
-  - TEST_TYPE='api' BLOCKCHAIN_TYPE='--blockchain-type=geth --blockchain-cache'
+  - TEST_TYPE='integration' BLOCKCHAIN_TYPE='--blockchain-type=geth' TEST_IGNORE='raiden/tests/integration/transfer,raiden/tests/integration/rpc'
```

Comment:
The TEST_IGNORE doesn't seem to work. E.g. here is a test from the transfer module being executed;

https://travis-ci.org/raiden-network/raiden/jobs/384760772#L1068

Reply:
Strange, it worked locally.

Reply:
it's probably a problem with how Travis substituted the variable

Reply:
https://docs.pytest.org/en/latest/example/pythoncollection.html?highlight=ignore#ignore-paths-during-test-collection

You should use `pytest --ignore=dir1 --ignore=dir2` instead of `pytest --ignore=dir1,dir2`